diagnostic.md

```diff
@@ -45,7 +59,23 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+.big {
+
+}
 
+#contentPane {
+
+}
+
+.important .red {
```

Comment:
This will select elements that have ```red``` as a class, that is inside of an element that has ```important``` as a class.  To select elements that have both ```important``` and ```red```, use ```.important.red```.